src/components/ReactTags.js

```diff
@@ -131,13 +131,16 @@ class ReactTags extends Component {
       return this.props.handleFilterSuggestions(query, suggestions);
     }
 
+    const queryIndex = (item) =>
+      item[this.props.labelField]
+        .toLowerCase()
+        .indexOf(query.toLowerCase());
     return suggestions.filter((item) => {
-      return (
-        item[this.props.labelField]
-          .toLowerCase()
-          .indexOf(query.toLowerCase()) === 0
-      );
-    });
+      return queryIndex(item) === 0;
+    })
+      .concat(suggestions.filter((item) => {
+        return queryIndex(item) > 0;
```

Comment:
both of this can be merged to `queryIndex(item) >= 0`

Reply:
I will talk about contries as example for explanation.
When we use "to" as query for search:
- in my version we get "**To**go, **To**ngo, Equa**to**rial Guinea, **Es**tonia, ...etc."
- in merged variant (>=0) we get "Equa**to**rial Guinea, **Es**tonia, ...etc., and only at the end **To**go, **To**ngo"
My opinion - first variant is more suitable, but the decision is yours.
What do you think?

Reply:
@Gibr I get your point. I agree with you. In that case can you make two variables, `exactSuggestions` and `partialSuggestions` and then return `exactSuggestions.concat(partialSuggestions)` so its more clear ?